calico/felix/actor.py

```diff
@@ -224,9 +135,8 @@ class Actor(object):
     max_ops_before_yield = 10000
     """Number of calls to self._maybe_yield before it yields"""
 
-    def __init__(self, queue_size=None, qualifier=None):
-        queue_size = queue_size or self.queue_size
-        self._event_queue = Queue(maxsize=queue_size)
+    def __init__(self, qualifier=None):
+        self._event_queue = Queue()
```

Comment:
@plwhite I've just made the queues unbounded.  It's simpler than the spawn approach, which was causing problems anyway.